cmake/developer_package/compile_flags/os_flags.cmake

```diff
@@ -333,6 +333,8 @@ if(WIN32)
         ie_add_compiler_flags(/Qdiag-disable:161,177,556,1744,1879,2586,2651,3180,11075,15335)
     endif()
 
+	ie_add_compiler_flags(/Qspectre)
```

Comment:
Please add explanation why do we need this flag

Reply:
Updated title and details. Is it enough?